Adds a small note on an OVN bug that is being carried over from 4.8 (#36626)
openshift/openshift-docs: release_notes/ocp-4-9-release-notes.adoc

// TODO: This known issue should carry forward to 4.9 and beyond!
* The `oc annotate` command does not work for LDAP group names that contain an equal sign (`=`), because the command uses the equal sign as a delimiter between the annotation name and value. As a workaround, use `oc patch` or `oc edit` to add the annotation. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1917280[*BZ#1917280*])

> * An Open Virtual Network (OVN) bug causes persistent connectivity issues with Octavia load balancers. When Octavia load balancers are created, OVN might not plug them into some Neutron subnets. These load balancers might be unreachable for some of the Neutron subnets. This problem affects Neutron subnets, which are created for each OpenShift namespace, at random when Kuryr is configured. As a result, when this problem occurs the load balancer that implements OpenShift `Service` objects will be unreachable from OpenShift namespaces affected by the issue. Because of this bug, {product-title} 4.8 deployments that use Kuryr SDN are not recommended on {rh-openstack-first} 16.1 with OVN and OVN Octavia configured. This bug will be fixed with the release of {rh-openstack} 16.1.7, which is currently scheduled for release on 2021-10-13. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1937392[*BZ#1937392*])

Reviewer: ```suggestion
* An Open Virtual Network (OVN) bug causes persistent connectivity issues with Octavia load balancers. When Octavia load balancers are created, OVN might not connect them to some Neutron subnets. These load balancers might be unreachable for some of the Neutron subnets. This problem randomly affects Neutron subnets, which are created for each OpenShift namespace, when Kuryr is configured. When this problem occurs, the load balancer that implements OpenShift `Service` objects will be unreachable from OpenShift namespaces that are affected by the issue. Because of this bug, {product-title} 4.8 deployments that use Kuryr SDN are not recommended on {rh-openstack-first} 16.1 with OVN and OVN Octavia configured. This bug will be fixed with the release of {rh-openstack} 16.1.7, which is currently scheduled for release on 2021-10-13. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1937392[*BZ#1937392*])
```
